lib/publiccloud/ec2.pm

```diff
@@ -28,7 +41,7 @@ sub find_img {
 sub create_keypair {
```

Comment:
I think we need to do one of two : 
1. move `create_keypair` into aws_client
2. keep ssh related things fully in ec2 

currently it felt like something solid split into two pieces ...

Reply:
What other ssh related things are outside `ec2.pm`?

Reply:
the field itself is in `aws_client` while code which init it is still here

Reply:
I prefer to remove ssh_key field from aws_client, the creation of keypair is only for ec2

Reply:
done